### ISSUE TITLE
Error on join

### ISSUE THREAD

**Issue by artur99** [SkyWars/SkyWars#183]
Any player would type /sw join, will get the default message telling him he will be added to quee.
But the game won't start...

```
While this, the console says this:
[17:22:07 ERROR]: Could not pass event AsyncPlayerChatEvent to SkyWars v2.0.0
org.bukkit.event.EventException
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:294) ~[craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g07
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g07d4
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:501) [craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g07
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:483) [craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g07
        at net.minecraft.server.v1_7_R4.PlayerConnection.chat(PlayerConnection.java:882) [craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g07
        at net.minecraft.server.v1_7_R4.PlayerConnection.a(PlayerConnection.java:832) [craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g07d45
        at net.minecraft.server.v1_7_R4.PacketPlayInChat.a(PacketPlayInChat.java:28) [craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g07d455
        at net.minecraft.server.v1_7_R4.PacketPlayInChat.handle(PacketPlayInChat.java:47) [craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g0
        at net.minecraft.server.v1_7_R4.NetworkManager.a(NetworkManager.java:84) [craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g07d4558-b3
        at net.minecraft.server.v1_7_R4.NetworkManager.channelRead0(NetworkManager.java:204) [craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24
        at net.minecraft.util.io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:98) [craftbukk
        at net.minecraft.util.io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) [
        at net.minecraft.util.io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) [cr
        at net.minecraft.util.io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:173) [craftbukkit.jar:
        at net.minecraft.util.io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) [
        at net.minecraft.util.io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) [cr
        at net.minecraft.util.io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:173) [craftbukkit.jar:
        at net.minecraft.util.io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) [
        at net.minecraft.util.io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) [cr
        at net.minecraft.util.io.netty.handler.timeout.ReadTimeoutHandler.channelRead(ReadTimeoutHandler.java:149) [craftbukkit.jar:gi
        at net.minecraft.util.io.netty.channel.DefaultChannelHandlerContext.invokeChannelRead(DefaultChannelHandlerContext.java:337) [
        at net.minecraft.util.io.netty.channel.DefaultChannelHandlerContext.fireChannelRead(DefaultChannelHandlerContext.java:323) [cr
        at net.minecraft.util.io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:785) [craftbukkit.ja
        at net.minecraft.util.io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:100) [craftbu
        at net.minecraft.util.io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:480) [craftbukkit.jar:git-Bukkit-
        at net.minecraft.util.io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:447) [craftbukkit.jar:g
        at net.minecraft.util.io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:341) [craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g
        at net.minecraft.util.io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:101) [craftbukki
        at java.lang.Thread.run(Unknown Source) [?:1.7.0_71]
Caused by: java.lang.NullPointerException
        at net.daboross.bukkitdev.skywars.listeners.ScoreReplaceChatListener.onChat(ScoreReplaceChatListener.java:39) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor384.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_71]
        at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_71]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:292) ~[craftbukkit.jar:git-Bukkit-1.7.9-R0.2-24-g07
        ... 28 more
```

**Comment by artur99** [comment]
The report: http://is.gd/cd2WRR

**Comment by FabioZumbi12** [comment]
Probally conflict with your chat plugin.

**Comment by daboross** [comment]
My apologies for absence for quite a period of time, I did abandon the project. I'm not fully back but I am doing some work.

This should be fixed in v2.0.1.
